### PR TITLE
Project1

### PR DESCRIPTION
**Pull request by divyalakshmi1011**
Project-1 was done.

### CODE REVIEW

File: Project1/example.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    emailID = request.form.get("emailID")
+    user_details.append(emailID)
+    if request.form.get("male") == "on" :
+        user_details.append("Male")
+    else :
+        user_details.append("Female")
+    if request.method == "GET" :
+        return render_template("Register.html")
+    if request.method == "POST" :
+        return render_template("Profile.html", user_details=user_details)
```
Comment (manasa133) [inline Project1/example.py]: How do you indicate registration is successfull or not

Comment (divyalakshmi1011) [reply Project1/example.py]: I have modified and completed Project1 and created another pull request. Please review.

**Comment by divyalakshmi1011** [comment]
Please review mam.

**Comment by manasa133** [comment]
Done with comments, you can merge

**Pull request closed without merging**